controllers/virtualnetwork_controller.go

```diff
@@ -15,7 +15,7 @@ type VirtualNetworkReconciler struct {
 }
 
 // +kubebuilder:rbac:groups=azure.microsoft.com,resources=virtualnetworks,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=azure.microsoft.com,resources=virtualnetworks/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=azure.microsoft.com,resources={virtualnetworks/status,virtualnetworks/finalizers},verbs=get;update;patch
```

Comment:
Do you need to regenerate the helm chart too, so that these changes get propagated into some manifest somewhere? Or is just changing this comment somehow magically enough?

Reply:
That's a good point! I'll add that to this PR.